docs/internals/contributing/bugs-and-features.txt

```diff
@@ -88,16 +90,18 @@ Requesting features
 We're always trying to make Django better, and your feature requests are a key
 part of that. Here are some tips on how to make a request most effectively:
 
-* Make sure the feature actually requires changes in Django's core. If your
+* Be aware that your feature may not require changes in Django's core. If your
   idea can be developed as an independent application or module — for
   instance, you want to support another database engine — we'll probably
   suggest that you develop it independently. Then, if your project gathers
   sufficient community support, we may consider it for inclusion in Django.
 
-* First request the feature on the `Django Forum`_, not in the ticket tracker.
-  It'll get read more closely and reach a larger audience. This is even more
-  important for large-scale feature requests. We like to discuss any big
-  changes to Django's core before actually working on them.
+* First request the feature on the `new feature ideas tracker`_, not in the
+  ticket tracker. This is where the community and Steering Council evaluate
+  whether we want to include features in core or encourage the creation of a
+  third party package. This is even more important for large-scale feature
+  requests. We like to discuss any big changes to Django's core before actually
+  working on them.
```

Comment:
```suggestion
  ticket tracker. This is where the community and Steering Council evaluate
  features for the Django eco-system. This is especially important for large-scale feature
  requests. We like to discuss any big changes to Django's core before actually
  working on them. We may also decide that a feature is best as a third-party
  package where it can be implemented outside of Django core's release cycle.
```

I think we should clarify that it's evaluating features as a whole rather than whether it's core or third-party. I adjusted some of the language as well, but I'm not tied to my suggestions. I eyeballed the line lengths, so that will need to be reformatted as well.